docs/windows.md

```diff
@@ -2,11 +2,142 @@
 
 ## Development Environment
 
-To setup a development environment in Windows:
-1) Install [Git for Windows](https://gitforwindows.org)
-1) Go to [MSYS2 website](http://www.msys2.org) and download the x86_64 installer
-1) Follow the instructions on the page for setting up the basic environment
-1) Run `C:\msys64\mingw64.exe` - a terminal window should pop up
+### Choco
+We recommend using [Chocolately](https://chocolatey.org/) as a package manager
+in Windows.
+
+To install it, open PowerShell as an administrator, then execute:
+
+```PowerShell
+Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+```
+
+### Git
+To setup a development environment in Windows first install
+[Git](https://gitforwindows.org) by executing as an adminstrator:
+
+```PowerShell
+choco install git
+```
+
+### MSYS2
+Both of the development environments in the next steps need MSYS2 installed.
+
+Install [MSYS2](http://www.msys2.org/):
+
+Keep PowerShell open as administrator and execute:
+```PowerShell
+choco install msys2
+```
+
+### GTK and Python with gvsbuild (recommended)
+gvsbuild provides a Python script helps you build the GTK library stack for
+Windows using Visual Studio. By compiling GTK with Visual Studio, we can then
+use a standard Python development environment in Windows.
+
+Note: gvsbuild says it requires msys2, but does it for our needs?
+
+First we will install the gvsbuild dependencies:
+1. Visual C++ build tools workload for Visual Studio 2019 Build Tools
+1. Python 3.6 (yes, this version is pretty old)
+
+#### Install Visual Studio 2019
+With your admin PowerShell terminal:
+
+```PowerShell
+choco install visualstudio2019-workload-vctools
+```
+
+#### Install Python 3.6
+
+1. Download [Python 3.6.8](https://www.python.org/ftp/python/3.6.8/python-3.6.8-amd64-webinstall.exe)
+1. Launch the installer by double clicking on it
+1. On the first screen of the installation wizard, leave the default options and
+select Customize installation
+1. On the Optional Features screen, select Next to keep the defaults
+1. On the Advanced Options screen, unselect the two selected options
+1. In the Customize install location field enter `C:\Python36` and select Install
+1. Allow the installation to finish
+
+#### Install gvsbuild
+We have forked gvsbuild for now because the upstream project is a little slow at
+accepting our Pull Requests. If we can get our changes upstream, then we'll
```

Comment:
"Slow" is not really constructive. Gaphor has slightly different requirements from the original gvsbuild.

Reply:
Good point, I wasn't trying to make this come off as unconstructive, I like your wording much better 👍